src/map/packets_struct.h

```diff
@@ -375,6 +375,11 @@ enum packet_headers {
 #else
 	guildLeave = 0x15a,
 #endif
+	moveToAttack = 0x139,
```

Comment:
this style of definition should not be used for new packets, please change to `DEFINE_PACKET_HEADER` as it does compile time validation of the sizes